frontend/public/style.scss

```diff
@@ -20,6 +17,12 @@
 @import 'style/mixin/truncate';
 @import 'style/mixin/line-clamp';
 
+// Ancillary Styles to support removing Bootstrap
+@import 'style/ancillary/bootstrap';
+
+// Ancillary Styles to achieve likeness with PatternFly4
+@import 'style/ancillary/patternfly4-ancillary';
```

Comment:
Nit:  does the file need to be called `patternfly4-ancillary` if its in the `ancillary` folder?  I notice you didn't do that above with `bootstrap`.

Reply:
renamed